pages/01-cartesian-chart/line-dash-styles.page.tsx

```diff
@@ -0,0 +1,97 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Shouldn't I see this new page in the visual regression snapshots?

Reply:
We already take that for all pages unless explicitly excluded. The new one is saved as `index-test-ts-matches-snapshot-for-01-cartesian-chart-line-dash-styles-screenshot-mode-true-1-snap.png`

Reply:
I see this message in the summary:

```
[test/visual/index.test.ts > matches snapshot for /#/01-cartesian-chart/line-dash-styles?screenshotMode=true: test/visual/index.test.ts#L24](https://github.com/cloudscape-design/chart-components/pull/114/files#annotation_40679642987)
Error: New snapshot was not written. The update flag must be explicitly passed to write a new snapshot.

 + This is likely because this test is run in a continuous integration (CI) environment in which snapshots are not written by default.


 ❯ test/visual/index.test.ts:24:25
 ❯ test/utils.ts:30:5
 ❯ node_modules/@cloudscape-design/browser-test-tools/use-browser.js:36:13
```

I guess when the old image is not present, the script fails differently and does not write a screenshot.